Dockerfile

```diff
@@ -1,23 +1,37 @@
-FROM openjdk:8-jdk
+FROM maven:3.6.3-jdk-8 as build
 
 ENV JAVA_OPTS "-server -Xconcurrentio -Xmx1g -Xms1g -XX:+UseG1GC -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"
 
-RUN mkdir -p /data && mkdir -p /graphhopper
-
 # install node - only required for JS UI
 RUN apt-get install -y wget \
        && curl -sL https://deb.nodesource.com/setup_13.x | bash - \
        && apt-get install -y nodejs
```

Comment:
Dropping the package cache might further reduce image size:

```sh
RUN apt-get install -y wget \
       && curl -sL https://deb.nodesource.com/setup_13.x | bash - \
       && apt-get install -y nodejs \
       && rm -rf /var/lib/apt/lists/*
```

Reply:
This is not part of the deployed image but it is part of the build image, which is not reflected in the total size.
The current image size is about 130 Mb, there's a way to reduce it if we use jre-slim but then we'll need to add wget to the image, which I'm not sure is so simple.
Also I guess that not all jar files are needed but probably just one which will reduce the image size significantly. But I'm not sure this is entirely related to this issue as the main purpose of this PR is to allow continuous images build.
I can create another PR and open anther issue if there's a need to reduce image size more.

Reply:
Sorry, I now see the name I gave to this pull request... :-/ In any case, the current changes are good enough on their own I believe. further improvement can be made later on assuming this will be tested to work as expected not just by me :-)

Reply:
> which I'm not sure is so simple.

See https://stackoverflow.com/a/28885237/194609

Reply:
My concern is not how simple it is to add wget to the image but rather what else is missing...
Since I don't know I prefer to start of like that and maybe later on reduce the size more when the whole process is up and running.
As I said, there are further improvements that can be made later on.